pkg/datapath/loader/loader.go

```diff
@@ -351,18 +351,32 @@ func (l *Loader) reloadDatapath(ctx context.Context, ep datapath.Endpoint, dirs
 	}
 
 	if ip := ep.IPv4Address(); ip.IsSet() {
+		scopedLog := ep.Logger(Subsystem).WithFields(logrus.Fields{
+			logfields.Veth: ep.InterfaceName(),
+		})
 		if ep.RequireEndpointRoute() {
-			upsertEndpointRoute(ep, *ip.IPNet(32))
+			if err := upsertEndpointRoute(ep, *ip.IPNet(32)); err != nil {
+				scopedLog.WithError(err).Warn("Failed to upsert route")
```

Comment:
Does this log message contain the actual route which could not be added/removed? That would be super helpful details to have when these cases are hit.